.github/workflows/regen-data.yml

```diff
@@ -1,9 +1,7 @@
-name: 'Regen Builds'
+name: 'Regen Data'
 
 on:
```

Comment:
This would probably be better being a cron ran something like hourly (or whatever keeps within the dokku deploy limits)

Reply:
Dokku is a self-hosted PaaS, so we don't really have any deploy limits other than the time it takes to do the deploys.

As for this kind of repo, I like the idea of using cron. I suppose we could start with once a day and see how it goes.